packages/garbo/src/fights.ts

```diff
@@ -509,6 +513,13 @@ export function dailyFights(): void {
             }
           }
 
+          if (
+            mallPrice($item`New Age healing crystal`) < expectedTargetProfit()
+          ) {
+            // Better to live and continue the chain than lose and not pprof fight at all
+            retrieveItem($item`New Age healing crystal`, count(fakeSources));
+          }
```

Comment:
```suggestion
          if (
            mallPrice($item`New Age healing crystal`) < expectedTargetProfit() &&
            have($familiar`pocket professor`) && !get("_garbo_weightChain", false)
          ) {
            retrieveItem($item`New Age healing crystal`, clamp(15 - get("_pocketProfessorLectures"), 0, 15);
          }
```

We don't have complex estimation for the number of profcopies we fight

Reply:
Then we'll buy 20 and hope for the best. The logs I've seen, there are 3 - 5 remaining in the chain, we start with 20 and if issues persist we raise it.

Reply:
> We don't have complex estimation for the number of profcopies we fight

It can be calculated once suitably attired for the chain
`ceil(square_root(familiar_weight(my_familiar()) + weight_adjustment() + my_familiar().soup_weight))`
is how I do it in ASH. Don't forget to add 2 for the memory chip.